### DISCUSSION TITLE
Feeling very stupid right now.

### DISCUSSION THREAD

**Discussion by CiberNin** [fastapi/fastapi#4608] [Questions] [answered]
So I'm looking at https://fastapi.tiangolo.com/ , but I don't see the actual documentation. Lots of very nice guides & tutorials yes, but where is the standard documentation of each function and it's arguments & their meaning?

**Comment by asaah18** [comment]
The codebase of fastapi is well-documented. So, all you need to have is a **good editor/IDE** to see documentation of specific function/class.

**Comment by Kludex** [comment, marked as answer]
Check https://github.com/tiangolo/fastapi/pull/2275.
